src/main/java/org/cascadebot/cascadebot/commands/subcommands/queue/QueueLoadSubCommand.java

```diff
@@ -0,0 +1,87 @@
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import org.cascadebot.cascadebot.utils.buttons.Button;
+import org.cascadebot.cascadebot.utils.buttons.ButtonGroup;
+
+import java.util.Set;
+
+public class QueueLoadSubCommand implements ICommandExecutable {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        if (context.getArgs().length < 1) {
+            context.getUIMessaging().replyUsage(this, "queue");
+            return;
+        }
+
+        context.getMusicPlayer().loadPlaylist(context.getArg(0), sender, (result, tracks) -> {
+            switch (result) {
+                case LOADED_GUILD:
+                case LOADED_USER:
+                    context.getUIMessaging().sendTracksFound(tracks);
+                    break;
+                case EXISTS_IN_ALL_SCOPES:
+                    ButtonGroup buttonGroup = new ButtonGroup(sender.getUser().getIdLong(), context.getChannel().getIdLong(), context.getGuild().getIdLong());
+                    buttonGroup.addButton(new Button.UnicodeButton(UnicodeConstants.ONE, ((runner, channel, message) -> {
+                        if (!runner.equals(buttonGroup.getOwner())) {
+                            return;
+                        }
+                        message.delete().queue();
+                        context.getMusicPlayer().loadPlaylist(context.getArg(0), sender, PlaylistType.USER, ((loadPlaylistResult, newTracks) -> {
+                            context.getUIMessaging().sendTracksFound(newTracks);
+                        }));
+                    })));
+                    buttonGroup.addButton(new Button.UnicodeButton(UnicodeConstants.TWO, ((runner, channel, message) -> {
+                        if (!runner.equals(buttonGroup.getOwner())) {
+                            return;
+                        }
+                        message.delete().queue();
+                        context.getMusicPlayer().loadPlaylist(context.getArg(0), sender, PlaylistType.GUILD, ((loadPlaylistResult, newTracks) -> {
+                            context.getUIMessaging().sendTracksFound(newTracks);
+                        }));
+                    })));
+                    context.getUIMessaging().sendButtonedMessage("Where you like to load this track from\n" + UnicodeConstants.ONE +
+                            " User\n" + UnicodeConstants.TWO + " Guild", buttonGroup);
+                    break;
+                case DOESNT_EXISTS:
+                    context.getTypedMessaging().replyDanger("Couldn't find playlist `" + context.getArg(0) + "`");
+                    break;
+            }
+        });
+    }
+
+    @Override
+    public String command() {
+        return "load";
+    }
+
+    @Override
+    public CascadePermission getPermission() {
+        return CascadePermission.of("Queue load sub command", "queue.load", true);
+    }
+
+    @Override
+    public String description() {
+        return null;
```

Comment:
Can you make this with @Deprecated(forRemoval=true) so I know to remove it for lang?